README.md

```diff
@@ -112,9 +107,12 @@ $ docker build -t registry.ng.bluemix.net/<ibmcloud_container_registry_namespace
 $ docker push registry.ng.bluemix.net/<ibmcloud_container_registry_namespace>/watsontesseract:1
 ```
 
-* Update the `image` in `watson-lang-trans.yml` with your image name
+* Update the `image` in `watson-lang-trans.yml` with your image name which in this case is `watsontesseract`.
```

Comment:
suggestion - "Update the `image` in `watson-lang-trans.yml` with your image name `watsonesseract`.